docs(site): add usage guidelines WO1 (#1019)
ec-europa/europa-component-library: src/website/src/pages/ec/components/list/docs/usage.mdx

---
title: Usage
order: 1
---

import { Paragraph } from '@ecl/website-components';

<Paragraph size="lead">

>   Lists group together related pieces of information so they are clearly

Reviewer: ```suggestion
Lists are used to present chunks of information in a concise and scannable way, lists help to organize content related to a single topic, grouping together a set of text.
```

>   associated with each other and easy to read. Lists are good from a structural

Reviewer: ```suggestion
```

>   point of view as they help create a well-structured and more accessible

Reviewer: ```suggestion

```

>   point of view as they help create a well-structured and more accessible.

Reviewer 2: "a well-structured and more accessible" what?

>   content. Semantically correct lists help visitors read your web site.

Reviewer: ```suggestion
```

</Paragraph>

## Anatomy

> Lists or "typography lists" are optimized for reading comprehension.

Reviewer: ```suggestion
Lists or "typography lists" are optimized for reading comprehension.
```
Lists are used to achieve easy scanability.

Reviewer: Bonjour, it is too slow to edit it line by line. When a long sentence is broken down to multiple lines...may I use the content but to edit it in a different page?

> Lists or "typography lists" are optimized for reading comprehension.

Reviewer: ```suggestion
  A list is usually organized vertically and it consists of a single continuous column of items displayed in rows. 
```

Author: you are mixing the anatomy and the do's and don't here (about nesting recommendation).
why "external link"?

> A list consists of a single continuous column of subdivisions called rows that contain items.

Reviewer: ```suggestion
```

> Content of each item may be plain text, or other elements (links, paragraphs, ...)

Reviewer: ```suggestion
```

> A list should have a contextual coherence and allow the user to understand the message instantly.

Reviewer: ```suggestion
```

Author: Why would we remove this? this is helpful content

> ### Nesting lists

Reviewer: ```suggestion
### Ordered list
```

Author: What?

> It is perfectly ok to nest one list (of any type) inside another one.

Reviewer: ```suggestion
```

Author: why would we remove this?

> You might want to have some sub-bullets sitting below a top level bullet.

Reviewer: ```suggestion
```

Author: why?

> Just make sure that nested items are all related to their parent.

Reviewer: ```suggestion
```

Author: why?

> Keep nested lists under control; having 4 or 5 nested level may create confusion and is generally a sign that the content has to be structured differently (break down into several lists for instance).

Reviewer: ```suggestion
```

Author: why?

> ### Three different types of lists

Reviewer: Please delete line 33-37

Author: why?

- unordered list - used to group a set of related items in no particular order
- ordered list - used to group a set of related items in a specific order
- description list - used to display name/value pairs such as terms and definitions; Each list type has a specific purpose and meaning in a web page.

> 

Reviewer: ```suggestion
Ordered lists display a set of items in a specific and logical order. They are commonly used as a step by step instructions.
Please note that different list type styles can be applied to list items, such as numbers, roman numerals or letters.

Recommendations:
- don't use punctuation such as commas, full stops or semicolons at the end of each point
However, if your bullet point is composed of more than 1 complete sentence, punctuation becomes necessary. Ask yourself whether your bullet point contains too much text.
```

#### Unordered (bulleted) Lists

> To present content when a set of items of equal status or value that can be placed in any order.

Reviewer: ```suggestion
Unordered lists are used to show content with the same importance, so the order of the items doesn't matter. Please note that different list type styles can be applied to bullets, like disc, circle, square or hyphen.
For ease of reading, we generally

- don't use a colon before the list
- don't capitalise the first letter of each point
- don't use punctuation such as commas, full stops or semicolons at the end of each point
- limit the numbers of items between 5 to 9
```

- Press contacts and accreditation
- Local offices of the EU
- List of departments
- Local offices outside the EU
- Find a staff member

#### Ordered (numbered) Lists

Imply sequence and order, and are commonly used when giving a set of instructions or to display a list of items that should be in a specific order.

##### Example

1. Eligibility criteria
2. Registration
3. Find partners
4. Prepare your proposal
5. Submission

If the list items were moved around into a different order, the information would no longer make sense:

1. Submission
2. Find partners
3. Eligibility criteria
4. Registration
5. Prepare your proposal

Ordered lists can be displayed with several sequencing options. The default in most browsers is decimal numbers, but there are others available:

- Letters
  - Lowercase ascii letters (a, b, c…)
  - Uppercase ascii letters (A, B, C…)
  - Lowercase classical Greek: (έ, ή, ί…)
- Numbers
  - Decimal numbers (1, 2, 3…)
  - Decimal numbers with leading zeros (01, 02, 03…)
  - Lowercase Roman numerals (i, ii, iii…)
  - Uppercase Roman numerals (I, II, III…)
  - Traditional Georgian numbering (an, ban, gan…)

>   - Traditional Armenian numbering (mek, yerku, yerek…)

Reviewer: Please delete line 43-91

#### Description lists

> Description lists are flexible; you can associate more than one value with a single name, or vice versa.

Reviewer: ```suggestion
Description lists contain definition or description, like a glossary or a list of speakers with relative biography.

Recommendations:

- try to use no more than one or two short sentences
- add a meaningful description or explanation

For example, the term “coffee” can have several meanings, and you could show them one after the other:

coffee

- a beverage made from roasted, ground coffee beans
- a cup of coffee
- a social gathering at which coffee is consumed
- a medium to dark brown color

Or, you can associate more than one name with the same value. This is useful to show variations of a term, all of which have the same meaning:

soda,
pop,
fizzy drink,
cola

- a sweet, carbonated beverage

> Note that description lists are not ordered.

Reviewer: Please delete line 43-115

> Note that description lists are not ordered.

Reviewer: Please delete line 97 to 115
